go.mod

```diff
@@ -102,7 +102,7 @@ require (
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.6-0.20210604193023-d5e0c0615ace
 	github.com/stackrox/external-network-pusher v0.0.0-20231115153210-b82d72f500a2
-	github.com/stackrox/helmtest v0.0.0-20230920075407-d83ac0b1e48e
+	github.com/stackrox/helmtest v0.0.0-20240116131801-96b6435a5cb3
```

Comment:
This is a commit on a PR https://github.com/stackrox/helmtest/pull/79

Reply:
Shall we merge this one with that reference?

Reply:
I'd like to merge helmtest first

Reply:
Updated